sam/methods.go

```diff
@@ -56,29 +56,26 @@ func (s SamSlice) Write(filename string) { // Does not write header
 	}
 }
 
-func (s *SamAln) WriteToFileHandle(file *fileio.EasyWriter) {
-	WriteAlnToFileHandle(file, s)
+func (s *Sam) WriteToFileHandle(file *fileio.EasyWriter) {
+	WriteToFileHandle(file, *s)
 }
 
-func (s *SamAln) NextRealRecord(file *fileio.EasyReader) bool {
+func (s *Sam) NextRealRecord(file *fileio.EasyReader) bool {
 	var done bool
-	var next *SamAln
-	for nextBytes, err := file.Peek(1); next == nil && !done; nextBytes, err = file.Peek(1) {
-		if err == nil && nextBytes[0] == '@' {
-			fileio.EasyNextLine(file)
-			continue
-		}
-		next, done = NextAlignment(file)
-	}
+	var next *Sam
```

Comment:
Not a bug, but it seems like next could be eliminated by changing line 73 to *s=curr